packages/paste-website/src/pages/components/spinner/index.mdx

```diff
@@ -0,0 +1,183 @@
+
+<Callout>
+  <CalloutTitle>Accessibility</CalloutTitle>
+  <CalloutText>
+    Please pay careful attention to our{' '}
+    <SiteLink to="/icon-system/#accessibility">icon accessibility guidelines</SiteLink>.
+  </CalloutText>
+</Callout>
+
+<LivePreview
+  scope={{
+    Spinner,
+  }}
+>
+  {`<Spinner decorative={false} title="Loading" />`}
+</LivePreview>
+
+## Sizing
+
+To resize the Spinner set a `size` prop. It will accept any valid [icon size token](/tokens/#sizings) as it's value. These are conveniently designed to be 1:1 to font sizing e.g. `font-size-40` can be paired with `icon-size-40`.
+
+<LivePreview
+  scope={{
+    Stack,
+    Spinner,
+  }}
+>
+  {`<Stack orientation="horizontal" spacing="space30">
+  <Spinner size="sizeIcon10" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon20" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon30" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon40" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon50" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon60" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon70" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon80" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon90" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon100" decorative={false} title="Loading" />
+  <Spinner size="sizeIcon110" decorative={false} title="Loading" />
+</Stack>`}
+</LivePreview>
+
+## Setting a color
+
+The Spinner, like any other icon, can be re-colored to meet your needs. Set an `iconColor` in the [same way you would with an icon](/icon-system/#adapting-the-icons-color).
+
+<LivePreview
+  scope={{
+    Stack,
+    Spinner,
+  }}
+>
+  {`<Spinner iconColor="colorTextSuccess" decorative={false} title="Loading" />`}
+</LivePreview>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/spinner - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {Spinner} from '@twilio-paste/spinner';
+
+const Component = () => {
+  return <Spinner decorative={false} title="Loading" />;
+};
+```
+
+#### Spinner Props
+
+##### `as?: keyof JSX.IntrinsicElements`
+
+Changes the sematic element of the spinner wrapper.
+
+##### `decorative?: boolean`
+
+Whether or not the SVG is just visual flair or adds meaning to the page. Specifically for screenreaders to know whether to read out the title or not.
+
+##### `title?: string`
+
+The accessibility text that is read when screenreaders get to this component.
+
+##### `size?: IconSize`
+
+Used to change the size of the Spinner.
+
+##### `iconColor?: TextColor`
+
+Sets the icon color to any of our text color tokens or currentColor, which inherits text color from the parent element.
+
+##### `delay?: number`
+
+Time delay in milliseconds of the animation.
```

Comment:
Should this show up in a table?

Reply:
a-lot of the new components are getting this format because the descriptions are getting pretty long, and the table is pretty cramped

Reply:
ahhh ok fair